frontend/app/views/spree/checkout/_confirm.html.erb

```diff
@@ -38,19 +37,39 @@
     </div>
     <dl>
       <div>
-        <dt><%= Spree.t(:shipping_address) %></dt>
+        <dt>
+          <%= Spree.t(:shipping_address) %>
+          <%= link_to spree.checkout_state_path('address'), class: 'align-text-bottom checkout-confirm-delivery-informations-link' do %>
```

Comment:
@aplegatt let's make a helper method of this - `checkout_edit_link('address')`
in `FrontendHelper`
```ruby
def checkout_edit_link(step = 'address')
  ...
end
```